--- /usr/local/lib/python3.4/dist-packages/sklearn/utils/testing.py	2014-08-01 07:24:18.879899483 +0000
+++ /usr/local/lib/python3.4/dist-packages/sklearn/utils/testing.py	2014-08-01 07:25:26.215900750 +0000
@@ -580,7 +580,9 @@
     """Safe way to reset warnings """
     warnings.resetwarnings()
     reg = "__warningregistry__"
-    for mod in sys.modules.copy().values():
+    for mod_name, mod in list(sys.modules.items()):
+        if 'six.moves' in mod_name:
+            continue
         if hasattr(mod, reg):
             getattr(mod, reg).clear()
